README.md

```diff
@@ -2,24 +2,12 @@
 
 ## Usage
 
-You can use both V1 and V2 style if you don't request multiport.
-
-### V1
-
-```
-GET /ormp/fee?from_chain_id=46&to_chain_id=1&from_address=0x65be094765731f394bc6d9df53bdf3376f1fc8b0&to_address=0x65be094765731f394bc6d9df53bdf3376f1fc8b0&payload=0x1841a477000000000000000000000000000000000000000000000000000000000000002e0000000000000000000000002b496f19a420c02490db859fefeccd71edc2c046000000000000000000000000dc0c760c0fb4672d06088515f6446a71df0c64c1000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000001a4e80f03d0000000000000000000000000000000000000000000000000000000000000002e000000000000000000000000e7578598aac020abfb918f33a20fad5b71d670b4000000000000000000000000092e19c46c9daab7824393f1cd9c22f5bea135600000000000000000000000004ca75992d2750bec270731a72dfdede6b9e71cc700000000000000000000000088a39b052d477cfde47600a7c9950a441ce61cb4000000000000000000000000000000000000000000000000000009184e72a000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000080000000000000000000000000c29dcb1f12a1618262ef9fba673b77140adc02d600000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000014000000000bb6a011db294ce3f3423f00eac4959e00000000000000000000000000000000000000000000000000000000000000000000000000000000&refund_address=0x570FCA2c6f902949dBb90664Be5680fEc94A84f6 HTTP/1.1
-Content-Type: application/x-www-form-urlencoded
-Host: g2.generic.darwinia.network:3378
-```
-
-### V2
-
 - Estimate fee of ormp
   
   ```
   POST /fee HTTP/1.1
```

Comment:
Is this the latest recommended request path? I remember there were some improvements to the path.